### PR TITLE
Decorate a class-OOP school library

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of the pull request

- [ ] Create a class Nameable
- [ ] make Person class inherit from Nameable
- [ ] create CapitalizeDecorator and TrimmerDecorator
- [ ] Create a class that inherits from BaseDecorator
- [ ] Modify ReadMe file

### CODE REVIEW

File: capitalize_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.capitalize
+    puts @nameable.correct_name.capitalize
```
Comment (mikethreels) [inline capitalize_decorator.rb]: - [ ] please remove the puts statement you included here for testing purposes.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kaweesi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: capitalize_decorator.rb (hunk 1 above)
Comment (Kaweesi-Matia) [reply capitalize_decorator.rb]: Remove puts statement

**Review by emmiiorji — APPROVED** [review]
Hi @Kaweesi-Matia

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Good commit messages
- Adhered to Gitflow
- Linter checks are passing
- Requested change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**